src/Illuminate/Validation/NotPwnedVerifier.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+namespace Illuminate\Validation;
+
+use Exception;
+use Illuminate\Contracts\Validation\NotCompromisedVerifier;
+use Illuminate\Http\Client\Factory;
+use Illuminate\Support\Str;
+
+class NotPwnedVerifier implements NotCompromisedVerifier
+{
+    /**
+     * The http factory instance.
+     *
+     * @var \Illuminate\Http\Client\Factory
+     */
+    protected $factory;
+
+    /**
+     * Create a new not compromised verifier.
+     *
+     * @param  \Illuminate\Http\Client\Factory  $factory
+     * @return void
+     */
+    public function __construct($factory)
+    {
+        $this->factory = $factory;
+    }
+
+    /**
+     * Verify that the given value has not been compromised in public breaches.
+     *
+     * @param  string  $value
+     * @return bool
+     */
+    public function verify($value)
+    {
+        if (empty($value = (string) $value)) {
+            return false;
+        }
+
+        [$hash, $hashPrefix] = $this->getHash($value);
+
+        return ! $this->search($hashPrefix)
+            ->contains(function ($line) use ($hash, $hashPrefix) {
+                [$hashSuffix, $count] = explode(':', $line);
+
+                return $hashPrefix.$hashSuffix == $hash && $count > 1;
```

Comment:
I'd suggest making the `$count` configurable, rather than hard-coding it to `1`. This will allow a sane threshold to be set depending on the application and technical proficiency of the users. As you may decide it makes more sense to allow passwords seen in a small number of breeches because blocking those users will cause them to give up in frustration. 

Even if you don't make it configurable, I'm curious why it's `1` and not `0`? At the moment, it will allow passwords that have been seen once in breeches through.

Reply:
This could be configured like this, for example setting a limit of 100:
```
'password' => Password::min(8)->ensureNotCompromised(100), 
```

Reply:
Just fixed the default compromised number from 1 to 0.

Reply:
Still investigating making the `$count` configurable.

Reply:
Awesome, thanks! 😁

Reply:
@valorin Can you let me know what do you think about https://github.com/laravel/framework/pull/36960/commits/a0fe44c2b09e78890f2d93d1c9c35b43bae00d92.